src/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -236,7 +236,7 @@ def execute(self):
         self.jobCount += 1
         try:
             jobReport.setJobParameter(par_name="MatcherServiceTime", par_value=str(matchTime), sendFlag=False)
-
+            jobReport.setJobParameter(par_name="JobType", par_value=jobType, sendFlag=False)
```

Comment:
The `JobWrapper` module is a better place from where this info should be reported.

Reply:
OK, in fact I wasn't sure from where it should be reported